### PR TITLE
Grab then equip

### PR DESCRIPTION
**Pull request by sethalves**
- allow near-grabbing an entity with one hand and then equipping it with the other

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
(1) near-grab (with the hydra trigger) an entity in one hand
(2) equip the entity (with the hydra bumper) in the other hand
(3) note that the entity didn't vanish
(4) release the grab (from the first hand)
(5) note that the entity stays equipped in the 2nd hand
(6) press the bumper in on the 2nd hand again, note that the entity is released

**Comment by hifi-gustavo** [comment]
Build console output for commit: c2cbca88cbc0dda409abc2f120523f96b1b29ed2 available at: https://gist.github.com/fcbc321cf3cfa3eaccd3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7057.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7057.dmg

**Comment by sethalves** [comment]
https://app.asana.com/0/26485730907942/88849821136140

**Comment by cozza13** [comment]
tested and works

File: libraries/entities/src/EntityItem.cpp [hunk 1]
```diff
@@ -1455,6 +1464,13 @@ void EntityItem::updateAngularVelocity(const glm::vec3& value) {
     }
 }
 
+void EntityItem::updateAngularVelocityFromNetwork(const glm::vec3& value) {
+    if (shouldSuppressLocationEdits()) {
+        return;
+    }
+    return updateAngularVelocity(value);
+}
```
Comment (AndrewMeadows) [inline libraries/entities/src/EntityItem.cpp]: Technically it compiles, but I had a little "Wtf?!!!" moment when I read it: this **void** function returns the value of another function... which happens to be **void**.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 49f2724608370d86c62ba6415079c4ad80e450cd available at: https://gist.github.com/fcbc321cf3cfa3eaccd3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7057.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7057.dmg

**Pull request merged**